lib/AArch64Base.ml

```diff
@@ -1290,7 +1290,7 @@ let do_pp_instruction m =
   | I_OP3 (v,op,r1,r2,kr, s) ->
       pp_rrkr (pp_op op) v r1 r2 kr ^ pp_barrel_shift "," s (m.pp_k)
   | I_ADDR (r,lbl) ->
```

Comment:
Thanks Luke. While you're at it, is it worth fixing this as well?

Reply:
I'm struggling to see what is wrong on line 1291 (I checked the parser and instruction set for a candidate instruction that matches this pattern [1] too), are you referring to the name of the instruction `I_ADDR`?

[1] https://developer.arm.com/documentation/dui0802/a/A64-General-Instructions/ADD--immediate-

Reply:
Yes I think `I_ADDR` has the same typo and ideally, it should be `I_ADR`